.changeset/mighty-years-own.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-react': patch
+---
+
+Move the `uiSchema` to its own property to align with component development and access of `ui:options`
```

Comment:
Probably mention that this applies to the next stuff

Reply:
Sure - will fix that up now